web/src/rendered_markdown.js

```diff
@@ -137,6 +139,22 @@ export const update_elements = ($content) => {
         }
     });
 
+    $content.find(".topic-mention").each(function () {
+        const $message_row = $content.closest(".message_row");
+        const message_id = rows.id($message_row);
+        const message = message_store.get(message_id);
+        const stream_id = message.stream_id;
```

Comment:
We can't rely on `message_row` existing -- consider drafts, message edit history, and compose previews.

I'll try to rework this.